portfolio/src/main/java/com/google/sps/servlets/HelloWorld.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
This comment seems incorrect since the servlet is annotated for "/welcome"

Reply:
Got it.